examples/oriented_images.html

```diff
@@ -0,0 +1,79 @@
+<html>
+    <head>
+        <title>Itowns - Oriented image</title>
+
+        <style type="text/css">
+            html {
+                height: 100%;
+            }
+
+            body {
+                margin: 0;
+                overflow:hidden;
+                height:100%;
+            }
+
+            #viewerDiv {
+                margin : auto auto;
+                width: 100%;
+                height: 100%;
+                padding: 0;
+            }
+            #menuDiv {
+                position: absolute;
+                top:0px;
+                margin-left: 0px;
+            }
+            #info {
+                color: black;
+                position: absolute;
+                bottom: 0;
+                right: 0;
+                margin-right: 15px;
+                padding: 0.3rem;
+                background-color: rgba(255, 255, 255, 0.493);
+                border: 2px solid black;
+                border-radius: 5px;
+            }
+            @media (max-width: 600px) {layer.orientation, 
+                #menuDiv {
+                    display: none;
+                }
+            }
+        </style>
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+        <script src="js/OrientedImageHelper.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv">
+            <p id="info">
+                Picture taken by Walter Mittelholzer in 1919, metadata from 
+                <a href="https://smapshot.heig-vd.ch/map/?imageId=2873" target="_blank"> smapshot.heig-vd.ch </a>
+            </p>
+        </div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            var renderer;
```

Comment:
This is not needed because you're not testing this example.
You can also remove the `oriented_image.js` file and move its content in this file.

Reply:
You should use the `css` introduced in PR #736